go.mod

```diff
@@ -4,26 +4,41 @@ go 1.12
 
 require (
 	github.com/benlaurie/objecthash v0.0.0-20180202135721-d1e3d6079fc1
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/coreos/bbolt v1.3.2 // indirect
```

Comment:
What caused all these changes? Should they be in a separate PR?

Reply:
see #1294